easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-intel-2018.01.eb

```diff
@@ -0,0 +1,26 @@
+# Built with EasyBuild version 3.5.3 on 2018-03-19_17-48-32
+name = 'FFTW'
+version = '3.3.6'
```

Comment:
@usman682 Why `3.3.6` when `3.3.7` is already available?

Reply:
We can try on 3.3.7 as well